ui/src/pages/backoffice/Patron/PatronDetails/components/ItemsCheckout/ItemsCheckout.js

```diff
@@ -17,7 +17,7 @@ export default class ItemsCheckout extends Component {
       <Loader isLoading={isLoading}>
         <Error error={error}>
           <Message color="blue">
-            Scan the barcode to perform a loan for this user.
+            Scan or type the barcode to perform a loan for this patron.
```

Comment:
```suggestion
            Scan or type the barcode to checkout a new loan for this patron.
```
WDYT? Or maybe to `create a new loan` as in the confirmation message?
I was thinking that maybe `to perform a loan` is not super clear?

Reply:
I went with `checkout physical copy`